lib/iris/coords.py

```diff
@@ -1578,6 +1578,28 @@ def bounds(self, bounds):
         self._bounds = bounds
 
 
+class CellMeasures(AuxCoord):
```

Comment:
I am intrigued that you chose to subclass `AuxCoord` rather than `Coord` directly. What was it that made you choose this approach?